tests/test_unique_queue.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+
+from preggy import expect
+
+from remotecv.unique_queue import UniqueQueue
+
+
+QUEUE = "queue"
+KEY = "key"
+
+class UniqueQueueTestCase(TestCase):
+    def test_should_detect_something(self):
+        key = UniqueQueue()._create_unique_key(QUEUE, KEY)
+        expect(key).to_equal('resque:unique:queue:queue:key')
```

Comment:
Nit: I may be a bit more clear to use some other strings like `foo` and/or `bar`. Also, would be great to add inputs to test `_escape_for_key` and also some input that shows the need for that `str`. WDYT?